src/etc/test-float-parse/src/gen/exhaustive.rs

```diff
@@ -0,0 +1,42 @@
+use std::fmt::Write;
+use std::ops::RangeInclusive;
+
+use crate::{Float, Generator, Int};
+
+/// Test every possible bit pattern. Not recommended for anything larger than `f32`!
```

Comment:
Is "not recommended" really the right phrasing? It's ~impossible for a 2^64 test to complete, and we don't have intermediate types like ~f48 or something.

Reply:
Good point, I updated the wording here